src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -15,6 +15,9 @@ public class ShooterConstants {
     public static final double MOMENT_OF_INERTIA_TOP = 0.08;
     public static final double MOMENT_OF_INERTIA_BOTTOM = 0.08;
 
+    public static final TalonFXConfiguration topMotorConfiguration = new TalonFXConfiguration();
+    public static final TalonFXConfiguration bottomMotorConfiguration = new TalonFXConfiguration();
```

Comment:
Remember to configure current limits and inversion.

Reply:
f578ea5